utils/src/main/java/software/amazon/awssdk/utils/builder/SdkBuilder.java

```diff
@@ -55,9 +54,8 @@
      * @return B the mutated builder instance
      */
     @SuppressWarnings("unchecked")
-    @ReviewBeforeRelease("Why does the user have to return a builder if they're just modifying it? "
-                         + "That makes this harder to use.")
-    default B apply(UnaryOperator<B> function) {
-        return function.apply((B) this);
+    default B apply(Consumer<B> function) {
```

Comment:
Should we rename this to accept to match the Consumer interface? Also no longer a function

Reply:
`builder.accept(b -> b.endpoint("foo"))` just doesn't look quite right to me. `apply` feels okay to me, since we're applying a mutation to the builder. I can't quite think of something better.

Renamed the parameter.